NasdaqDataLink.cs

```diff
@@ -57,13 +59,26 @@ static NasdaqDataLink()
             // The NasdaqDataLink API now requires TLS 1.2 for API requests (since 9/18/2018).
             // NET 4.5.2 and below does not enable this more secure protocol by default, so we add it in here
             ServicePointManager.SecurityProtocol |= SecurityProtocolType.Tls12;
+
+            var potentialNasdaqToken = Config.Get("nasdaq-auth-token");
+            var potentialQuandlToken = Config.Get("quandl-auth-token");
+
+            if (potentialNasdaqToken.Length != 0)
```

Comment:
- Should use `string.IsNullOrEmpty()` instead, else default value for string will be `null` and throw
- Suggest directly moving `Config.Get("quandl-auth-token")` into the `else if` bellow, so that in the happy case it doesn't log anything (config.get will log if no value found)